client/Assets/Scripts/UI/UIModelManager.cs

```diff
@@ -116,8 +93,9 @@ private bool AnimationHasParameter(string parameterName)
         return param != null;
     }
 
-    public void HandleAnimation(string animationName, bool hasAnimationParameter)
+    public void HandleAnimation(string animationName)
     {
+        bool hasAnimationParameter = AnimationHasParameter(animationName);
```

Comment:
what i meant was to delete the `AnimationHasParamerter` method, the end result would be something like:
```suggestion
        bool hasAnimationParameter = modelAnimator
            .parameters
            .ToList()
            .Any(p => p.name == parameterName);
```